packages/core/src/workflows/workflow.test.ts

```
@@ -10,10 +10,11 @@
~import { createTool, Mastra } from '..';
~import { Agent } from '../agent';
~import { RequestContext } from '../di';
~import { MastraError } from '../error';
 import { MockStore } from '../storage/mock';
 import type { ChunkType, StreamEvent, WatchEvent } from './types';
 import { cloneStep, cloneWorkflow, createStep, createWorkflow, mapVariable } from './workflow';
+import { MastraError } from '../error';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'MastraError'. Consider to delete it or rename it.

'MastraError' is defined here:


(lint/suspicious/noRedeclare)